pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "torch_points3d"
-version = "1.4.0" # This will be overriden by the CI at publish time
+version = "1.4.3" # This will be overriden by the CI at publish time
```

Comment:
How does the publish mechanism work? Do you mean the version here bumped during the publish by the CI?

Reply:
No I set the version here myself, and this is the version as shown on our pypi server. I'm not exactly sure what the comment here refers to. Perhaps @pevers knows?